app-office/drawio-desktop-bin/drawio-desktop-bin-22.1.2.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CHROMIUM_LANGS="
+	af am ar bg bn ca cs da de el en-GB en-US es es-419 et fa fi fil fr gu he
+	hi hr hu id it ja kn ko lt lv ml mr ms nb nl pl pt-BR pt-PT ro ru sk sl sr
+	sv sw ta te th tr uk ur vi zh-CN zh-TW
+"
+
+inherit chromium-2 desktop unpacker xdg
+
+APP_NAME="drawio"
+
+DESCRIPTION="draw.io diagramming and whiteboarding desktop app"
+HOMEPAGE="https://www.drawio.com/"
+
+SRC_URI="
+	amd64? ( https://github.com/jgraph/drawio-desktop/releases/download/v${PV}/drawio-amd64-${PV}.deb
+	-> ${PN}-amd64-${PV}.deb )
+	arm64? ( https://github.com/jgraph/drawio-desktop/releases/download/v${PV}/drawio-arm64-${PV}.deb
+	-> ${PN}-arm64-${PV}.deb )
+	https://github.com/jgraph/drawio-desktop/blob/591f5169dbb2b1e6bcf9e7dffb0228423c32487f/build/icon.svg
+	-> drawio.svg
+"
+S="${WORKDIR}"
+
+KEYWORDS="-* ~amd64 ~arm64"
```

Comment:
No, I don't have the hardware. I would keep it in the package, just waiting for someone who can test it.

Reply:
The policy is to request keywording on bugzilla afterwards if you cant test it yourself. 

(This doesn't mean requesting keywording on every arch under the sun of course, too much work so someone actually using it would be good.)

Reply:
Can I leave the arch-conditional distfile in SRC_URI if I remove the ~arm64 keyword?

Reply:
You can, it'll just be there ready for the keywording then.